flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/catalog/hive/util/HiveTableUtil.java

```diff
@@ -421,6 +424,31 @@ public static void checkAcidTable(CatalogTable catalogTable, ObjectPath tablePat
 		}
 	}
 
+	/**
+	 * Returns a new Hadoop Configuration object using the path to the hadoop conf configured.
+	 *
+	 * @param hadoopConfDir Hadoop conf directory path.
+	 * @return A Hadoop configuration instance.
+	 */
+	public static Configuration getHadoopConfiguration(String hadoopConfDir) {
+		Configuration hadoopConfiguration = new Configuration();
+		if (new File(hadoopConfDir).exists()) {
+			if (new File(hadoopConfDir + "/core-site.xml").exists()) {
+				hadoopConfiguration.addResource(new Path(hadoopConfDir + "/core-site.xml"));
+			}
+			if (new File(hadoopConfDir + "/hdfs-default.xml").exists()) {
+				hadoopConfiguration.addResource(new Path(hadoopConfDir + "/hdfs-default.xml"));
+			}
+			if (new File(hadoopConfDir + "/hdfs-site.xml").exists()) {
+				hadoopConfiguration.addResource(new Path(hadoopConfDir + "/hdfs-site.xml"));
+			}
+			if (new File(hadoopConfDir + "/mapred-site.xml").exists()) {
```

Comment:
We also need yarn-site.xml, which is needed to generate Parquet splits in a kerberized environment.